docs/api/README.md

```diff
@@ -6,8 +6,10 @@ Using **gRPC** is preferred for better performance.
+| Insert Service | Insert new vector(s) into the Vald Agent Pods                                                                                                   | [Insert](../api/insert.md#insert-rpc)<br>[StreamInsert](../api/insert.md#streaminsert-rpc)<br>[MultiInsert](../api/insert.md#multiinsert-rpc)                                                                                                                                                                                                                                                  | [Vald Insert APIs](../api/insert.md) |
+| Update Service | Update the exists vector(s) in the Vald Agent Pods                                                                                              | [Update](../api/update.md#update-rpc)<br>[StreamUpdate](../api/update.md#streamupdate-rpc)<br>[MultiUpdate](../api/update.md#multiupdate-rpc)                                                                                                                                                                                                                                                  | [Vald Update APIs](../api/update.md) |
+| Upsert Service | Update the exists vector(s) in the Vald Agent Pods or Insert new vector(s) into the Vald Agent Pods if the vector is not exists                 | [Upsert](../api/upsert.md#upsert-rpc)<br>[StreamUpsert](../api/upsert.md#streamupsert-rpc)<br>[MultiUpdate](../api/upsert.md#multiupsert-rpc)                                                                                                                                                                                                                                                  | [Vald Upsert APIs](../api/upsert.md) |
+| Search Service | Search nearest neighbor vectors using query (vectors or IDs)<br>There are 2 types of Search methods: Search (means ANN Search) and LinearSearch | [Search](../api/search.md#search-rpc)<br>[SearchByID](../api/search.md#searchbyid-rpc)<br>[StreamSearch](../api/search.md#streamsearch-rpc)<br>[StreamSearchByID](../api/search.md#streamsearchbyid-rpc)<br>[MultiSearch](../api/search.md#multisearch-rpc)<br>[MultiSearchByID](../api/search.md#multisearchbyid-rpc)                                                                         | [Vald Search APIs](../api/search.md) |
+|       ^        | ^                                                                                                                                               | [LinearSearch](../api/search.md#linearsearch-rpc)<br>[LinearSearchByID](../api/search.md#linearsearchbyid-rpc)<br>[StreamLinearSearch](../api/search.md#streamlinearsearch-rpc)<br>[StreamLinearSearchByID](../api/search.md#streamlinearsearchbyid-rpc)<br>[MultiLinearSearch](../api/search.md#multilinearsearch-rpc)<br>[MultiLinearSearchByID](../api/search.md#multilinearsearchbyid-rpc) | ^                                    |
```

Comment:
I think we can merge it  to the `Search Service`, why did you separate from there?

Reply:
cuz, the API NAMES cell will be too long and it is not readable.
I used cell merge notation. ( it  does not work on Github, but it will work on the public document)